pd/src/staking.rs

```diff
@@ -0,0 +1,29 @@
+use tendermint::{vote, PublicKey};
+
+/// Validator tracks the Penumbra validator's long-term consensus key (tm_pubkey), as well as their
+/// voting power.
+#[derive(Debug, Eq, PartialOrd, Ord)]
+pub struct Validator {
+    tm_pubkey: PublicKey,
+    voting_power: vote::Power,
+}
```

Comment:
non-blocking on this PR, but in the future we might want to merge this into types in https://rustdoc.penumbra.zone/main/tendermint/validator/index.html (and/or modify them to suit our purposes; the `tendermint-rs` API isn't totally fleshed out yet)

Reply:
I wasn't sure if we wanted to have a separate type specifically for Penumbra validators: right now this type is the same as the Tendermint type, but in the future I was imagining that we may want to have more penumbra-specific data or methods in this struct

Reply:
Yeah, starting with our own type is a good call, just wanted to make the connection for the future.